### PR TITLE
OSDOCS 7779: mobb migration of WAF docs for ROSA (three assemblies)

### PR DESCRIPTION
**Pull request by ousleyp**
Version(s): 4.13+

Issue: [OSDOCS-7779](https://issues.redhat.com//browse/OSDOCS-7779)

Link to docs preview: 
* https://65073--docspreview.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/rosa-mobb-using-waf-rosa-osd-ocp-on-aws
* https://65073--docspreview.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/rosa-mobb-using-cloudfront-and-waf
* https://65073--docspreview.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/rosa-mobb-using-alb-and-waf

QE review: N/A
- [ ] QE has approved this change.

Additional information: migrating two individual pages + a page that links to both.

### CODE REVIEW

File: rosa_tutorials/rosa-mobb-using-alb-and-waf.adoc [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+// ---
+// date: '2021-06-17'
+// title: AWS ALB
+// aliases: ['/docs/aws/waf/cloud-front.md']
+// tags: ["AWS", "ROSA", "OSD"]
+// authors:
+//  - 'Connor Wooley'
+// ---
+
+You can use an Application Load Balancer (ALB) solution to add a Web Application Firewall (WAF) to your ROSA workloads. Using an external solution protects OpenShift resources from experiencing Denial of Service due to handling the WAF.
```
Comment (tmalove) [inline rosa_tutorials/rosa-mobb-using-alb-and-waf.adoc:20]:
```suggestion
You can use an Application Load Balancer (ALB) solution to add a Web Application Firewall (WAF) to your ROSA workloads. Using an external solution protects OpenShift resources from experiencing denial of service due to handling the WAF.
```

File: rosa_tutorials/rosa-mobb-using-alb-and-waf.adoc [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+----
+
+. Apply the CRDs to the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc apply -k "github.com/aws/eks-charts/stable/aws-load-balancer-controller//crds?ref=master"
+----
+
+. Add the Helm repo by running the following command:
```
Comment (tmalove) [inline rosa_tutorials/rosa-mobb-using-alb-and-waf.adoc:169]:
```suggestion
. Add the Helm repository by running the following command:
```

File: rosa_tutorials/rosa-mobb-using-cloudfront-and-waf.adoc [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+// ---
+// date: '2021-06-17'
+// title: Using CloudFront + WAF
+// aliases: ['/docs/aws/waf/cloud-front.md']
+// tags: ["AWS", "ROSA"]
+// authors:
+//  - 'Connor Wooley'
+// ---
+
+You can use CloudFront and a custom domain to add a Web Application Firewall (WAF) to your ROSA workloads. Using an external solution protects OpenShift resources from experiencing Denial of Service due to handling the WAF.
```
Comment (tmalove) [inline rosa_tutorials/rosa-mobb-using-cloudfront-and-waf.adoc:20]:
```suggestion
You can use CloudFront and a custom domain to add a Web Application Firewall (WAF) to your ROSA workloads. Using an external solution protects OpenShift resources from experiencing denial of service due to handling the WAF.
```

File: rosa_tutorials/rosa-mobb-using-cloudfront-and-waf.adoc [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+. View your WAF by running the following command:
++
+[source,terminal]
+----
+$ aws wafv2 list-web-acls --scope REGIONAL --region us-east-2 | jq .
+----
+
+. Add a certificate to ACM. 
+.. Navigate to https://us-east-2.console.aws.amazon.com/acm/home?region=us-east-1#/importwizard/.
+.. Paste the certificate, key, and certchain from the `certbot` files.
```
Comment (tmalove) [inline rosa_tutorials/rosa-mobb-using-cloudfront-and-waf.adoc:136]:
```suggestion
.. Paste the certificate, key, and certificate chain from the `certbot` files.
```
I think we are implementing changes like this in the second round, but want to flag it.

File: rosa_tutorials/rosa-mobb-using-cloudfront-and-waf.adoc [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+====
+You can view the endpoint by running the `$ oc get manageddomains` command.
+====
+* *Origin Protocol Policy*: `HTTPS only`
+* *Viewer Protocol Policy*: `Redirect HTTP to HTTPS`
+* *Allowed HTTP Methods*: `GET, HEAD, OPTIONS, PUT, POST, PATCH, DELETE`
+* *AWS WAF Web ACL*: `demo-waf-acl`
+* *Alternate Domain Names*: `*.<your_domain>`
+* *Custom SSL Certificate*: `<your_imported_certificate>`
+* For *Origin Request Policy*, create a new policy whitelist that includes `origin, user-agent, referer, host` (**IMPORTANT**)
```
Comment (tmalove) [inline rosa_tutorials/rosa-mobb-using-cloudfront-and-waf.adoc:158]:
```suggestion
* For *Origin Request Policy*, create a new policy allowlist that includes `origin, user-agent, referer, host` (**IMPORTANT**)
```

File: rosa_tutorials/rosa-mobb-using-cloudfront-and-waf.adoc [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+
+. Click *Create* and then wait until the *Status* is *Ready*.
```
Comment (tmalove) [inline rosa_tutorials/rosa-mobb-using-cloudfront-and-waf.adoc:160]:
```suggestion
. Click *Create* and wait until the *Status* is *Ready*.
```

**Review by tmalove — APPROVED** [review]

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Comment by ousleyp** [comment]
@michaelryanmcneill, please take a look at this when you can! Thanks so much.

**Comment by bmcelvee** [comment]
Thank you so much for all of the work on this! It's being picked up and continued in https://github.com/openshift/openshift-docs/pull/65514.

**Pull request closed without merging**